classes/deactivate.php

```diff
@@ -16,20 +16,26 @@ class Object_Sync_Sf_Deactivate {
 
 	protected $wpdb;
 	protected $version;
+	protected $slug;
+	protected $option_prefix;
+	protected $schedulable_classes;
 
 	/**
 	* Constructor which sets up deactivate hooks
 	* @param object $wpdb
 	* @param string $version
 	* @param string $slug
+	* @param string $option_prefix
 	* @param array $schedulable_classes
 	*
 	*/
-	public function __construct( $wpdb, $version, $slug, $schedulable_classes ) {
+	public function __construct( $wpdb, $version, $slug, $option_prefix, $schedulable_classes ) {
```

Comment:
Since `__construct` is a public method and this is the 1.4 release (not a 2.0 backwards compatibility breaking release), the new parameters should be optional. We can't guarantee developers haven't invoked this method. (As unlikely as that is for a plugin deactivation class.)